R/RtGam.R

```diff
@@ -48,15 +52,23 @@
 #' @return Stub function: NULL
 #' @export
 #' @examples
-#' cases <- c(1L, 2L, 3L)
-#' reference_date <- as.Date(c("2023-01-01", "2023-01-02", "2023-01-03"))
-#' mod <- RtGam::RtGam(cases, reference_date)
+#' withr::with_seed(12345, {
+#'   cases <- rpois(20, 10)
+#' })
+#' reference_date <- seq.Date(
+#'   from = as.Date("2023-01-01"),
+#'   length.out = 20,
+#'   by = "day"
+#' )
+#' fit <- RtGam::RtGam(cases, reference_date)
```

Comment:
kind of unusual to use package::function in your own package examples?

Reply:
https://github.com/CDCgov/cfa-gam-rt/pull/22/commits/a8fa503c606ec9facb3814ed0de4f34284d6d995